<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BlockModDemoDialog</class>
 <widget class="QDialog" name="BlockModDemoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1273</width>
    <height>663</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BlockMod Library Demo/Test</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QToolButton" name="toolButtonNew">
        <property name="icon">
         <iconset resource="../resources/BlockModDemo.qrc">
          <normaloff>:/gfx/filenew_32x32.png</normaloff>:/gfx/filenew_32x32.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButtonOpen">
        <property name="icon">
         <iconset resource="../resources/BlockModDemo.qrc">
          <normaloff>:/gfx/fileopen_32x32.png</normaloff>:/gfx/fileopen_32x32.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButtonSave">
        <property name="icon">
         <iconset resource="../resources/BlockModDemo.qrc">
          <normaloff>:/gfx/filesave_32x32.png</normaloff>:/gfx/filesave_32x32.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="toolButtonInfo">
        <property name="icon">
         <iconset resource="../resources/BlockModDemo.qrc">
          <normaloff>:/gfx/info_32x32.png</normaloff>:/gfx/info_32x32.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Network model</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="BLOCKMOD::ZoomMeshGraphicsView" name="graphicsView"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="pushButtonRemoveConnection">
          <property name="text">
           <string>Remove connection</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonRemoveBlock">
          <property name="text">
           <string>Remove block</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonAddBlock">
          <property name="text">
           <string>Add block</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Block editor</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" rowspan="3">
       <widget class="BLOCKMOD::ZoomMeshGraphicsView" name="blockEditorView"/>
      </item>
      <item row="0" column="1">
       <widget class="QToolButton" name="toolButtonAddSocket">
        <property name="icon">
         <iconset resource="../resources/BlockModDemo.qrc">
          <normaloff>:/gfx/plus.png</normaloff>:/gfx/plus.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QToolButton" name="toolButtonRemoveSocket">
        <property name="icon">
         <iconset resource="../resources/BlockModDemo.qrc">
          <normaloff>:/gfx/minus.png</normaloff>:/gfx/minus.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>261</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="checkBoxSocketIsInlet">
        <property name="text">
         <string>Inlet socket</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BLOCKMOD::ZoomMeshGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>BM_ZoomMeshGraphicsView.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/BlockModDemo.qrc"/>
 </resources>
 <connections/>
</ui>
